.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python: 
```

Comment:
Not sure why or how this change and the build health reporting in README are shown as missing in master (?)